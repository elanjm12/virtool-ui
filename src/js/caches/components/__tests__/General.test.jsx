import CacheGeneral from "../General";

describe("<CacheGeneral />", () => {
    it("should render", () => {
        const props = {
            hash: "foo",
            program: "bar"
        };
        const wrapper = shallow(<CacheGeneral {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
