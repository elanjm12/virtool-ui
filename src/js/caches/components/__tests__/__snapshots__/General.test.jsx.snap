// Vitest Snapshot v1

exports[`<CacheGeneral /> > should render 1`] = `
<styled.table>
  <tbody>
    <tr>
      <th>
        Hash
      </th>
      <td>
        foo
      </td>
    </tr>
    <tr>
      <th>
        Trimming Program
      </th>
      <td>
        bar
      </td>
    </tr>
  </tbody>
</styled.table>
`;
